pages/awards/index.js

```diff
@@ -26,18 +27,24 @@ import styles from '@styles/pages/Awards.module.scss'; // Page styles
 import useSnackbar from '@utils/useSnackbar'; // Snackbar
 
 const PAGE_SIZE = 20; // Rows per page
+const AWARDS_TABLE_HIDDEN_COLUMN = ['categories'];
```

Comment:
Have to add the column to filter in the actual table otherwise React Table does not run filter on it. So I decided to just add it as a column and hide it